cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -283,6 +273,11 @@ public void testGenerateDeploymentWithTls() {
 
         assertThat(io.strimzi.operator.cluster.TestUtils.containerEnvVars(containers.get(0)).get(KafkaBridgeCluster.ENV_VAR_KAFKA_BRIDGE_TRUSTED_CERTS), is("my-secret/cert.crt;my-secret/new-cert.crt;my-another-secret/another-cert.crt"));
         assertThat(io.strimzi.operator.cluster.TestUtils.containerEnvVars(containers.get(0)).get(KafkaBridgeCluster.ENV_VAR_KAFKA_BRIDGE_TLS), is("true"));
+
+        ConfigMap configMap = kbc.generateBridgeConfigMap(metricsAndLogging);
+        assertThat(configMap.getData().get(KafkaBridgeCluster.BRIDGE_CONFIGURATION_FILENAME), containsString("kafka.ssl.truststore."));
+        assertThat(configMap.getData().get(KafkaBridgeCluster.BRIDGE_CONFIGURATION_FILENAME), not(containsString("kafka.ssl.keystore.")));
+        assertThat(configMap.getData().get(KafkaBridgeCluster.BRIDGE_CONFIGURATION_FILENAME), containsString("kafka.security.protocol=SSL"));
```

Comment:
Why should it contain / not contain these values?

Reply:
Because the test is enabling TLS (with trusted certificates) on the bridge so I am verifying that the corresponding generated configuration contains what to expect (not to expect, i.e. keystore because no TLS authentication configured).

Reply:
Would it make more sense to have the whole file here once and otherwise rely on the unit tests of the configuration builder?

Reply:
Yeah I will just remove this checks because we already test these things in the unit test. Also, before the changes this part was just checking one TLS env vars so it was anyway not checking the full configuration.